src/redux/books/books.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState: [],
+  reducers: {
+    addBook: (state, action) => {
+      state.push(action.payload);
+    },
+    removeBook: (state, action) =>
+      // eslint-disable-next-line implicit-arrow-linebreak
+      state.filter((book) => book.id !== action.payload),
+  },
+});
+
+export const { addBook, removeBook } = booksSlice.actions;
+
```

Comment:
Great job using `createSlice` for writing your reducer as suggested by `redux-toolkit`, but please consider the following required changes:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Define%20action%20types%20for%20adding%20and%20removing%20a%20book.), Please define action types for adding and removing a book using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).
- [ ] Kindly delete the `eslint` disablement line 11 and fix it yourself.